Makefile

```diff
@@ -150,6 +150,7 @@ $(AUTOLOADS): $(ELFILES)
 	$(BATCH) \
 		--eval '(setq make-backup-files nil)' \
 		--eval "(setq generated-autoload-file (concat command-line-default-directory \"/\" \"$@\"))" \
+		--eval "(require 'autoload)" \
```

Comment:
`autoload.el` will be obsoleted since Emacs 29. So it is necessary to load explicitly on latest Emacs.